_docs_operate/configuration.md

```diff
@@ -607,6 +607,30 @@ This module is deprecated in favor of the [Message Broker Publisher](#messagebro
 
 You can find type definitions of the event data in the [Connector Events]({% link _docs_integrate/connector-events.md %}) section.
 
+#### sse (sever sent events) <a href="{% link _docs_operate/modules.md %}#sse"><i class="fas fa-fw fa-info-circle"/></a> {#sse}
+
+**Sample Configuration:**
+
+```jsonc
+{
+  // ...
+
+  "modules": {
+    "sse": {
+      "baseUrlOverride": "https://example.com"
```

Comment:
please don't document this. This is only for development purposes and should not be used.